sandbox/main.qml

```diff
@@ -0,0 +1,112 @@
+import QtQuick 2.14
+import QtQuick.Window 2.14
+import QtQuick.Controls 2.14
+
+Window {
+    id: rootWindow
+    width: 640
+    height: 480
+    visible: true
+    title: qsTr("Status App Sandbox")
+
+    ButtonGroup {
+        id: topicsGroup
+        buttons: tabs.children
+    }
+
+    Flow {
+        id: tabs
+        width: parent.width
+        Button {
+            checkable: true
+            text: "Text Styles"
+        }
```

Comment:
I don't believe it make sense to have this one as there most likely won't be multiple text style components (probably just `StatusText` which can then be adjusted in size and weight).